oas/swagger.yml

```diff
@@ -146,9 +147,9 @@ paths:
           content:
             application/json:
               schema:
-                additionalProperties: false # WARNING! without this, a consumer can add stuff that doesn't exist! See also https://bitbucket.org/atlassian/swagger-mock-validator/issues/84/test-incorrectly-passes-when-mock-expects
                 type: array
                 items:
+                  additionalProperties: false
```

Comment:
Is this the right spot? Shouldn't it be under the `schema` property?

Reply:
It's because I'm not targeting the responses arrays exclusively, just picking up the 6 occurrences where `type: object` exists. Could do will some more `sed` foo

Reply:
It doesn't fail validation on the swagger editor, but will test it out 👍🏾